CPAC/registration/utils.py

```diff
@@ -12,7 +12,7 @@ def seperate_warps_list(warp_list, selection):
 
 
 def hardcoded_reg(anatomical_brain, reference_brain, anatomical_skull,
-                  reference_skull):
+                  reference_skull, fixed_image_mask="<undefined>"):
```

Comment:
Since Python has the type `None` to represent undefined values, wouldn't be the case to use it here? `fixed_image_mask=None`

and check `if fixed_image_mask is not None:`

Reply:
Oh, it's just because I am stupid. I wanted to check what is returned when a connected node does not provide its input and use it in that test. But I forgot to modify it. Do you know what happens when I do 
`calc_ants_warp_wf.connect(inputspec, 'fixed_image_mask',
            calculate_ants_warp, 'fixed_image_mask')`
but calculate_ants_warp.inputs.fixed_image_mask doesn't have a value? Is the input is set at None?

Reply:
Yes! It will be `None`. But it will only work for arguments that have a default value. Otherwise, I guess it will throw an error, not sure.